README.md

```diff
@@ -375,6 +375,17 @@ transloco-keys-manager find --translations-path my/path
 transloco-keys-manager find -p my/path
 ```
 
+- `getTextMarker`: The marker sign for dynamic values only for extract key: (defaults to `marker`)
+
+```
+# for work need install and import marker function in source code
+# import { marker } from '@biesbjerg/ngx-translate-extract-marker';
```

Comment:
We definitely don't want this as an external library that transloco users need to install an ngx-translate lib. this should be part of the keys manager.

Reply:
no, transloco-keys-manager - devdeps, in prod this code not exists, if function place to transloco-keys-manager on production we have errors of not exists functions, it needs place to @ngneat/transloco-utils or other not devdeps

I added todo in a comment for it https://github.com/ngneat/transloco-keys-manager/pull/60/files#diff-27f79eb36e9a93d417cf6bce12c9aca0R16

Reply:
What you are saying isn't correct, if I'm using in my code function from a package that is installed in my `devDependecies` the build won't remove this code. Webpack is checking the code usage and not where the package is installed.

This marker function should be part of the transloco keys manager package.

Reply:
done

Reply:
done